release/apparmor.go

```diff
@@ -133,3 +143,55 @@ func AppArmorFeatures() []string {
 	}
 	return appArmorFeatures
 }
+
+// parser probe related code
+type apparmorParserFeature struct {
+	feature string
+	rule    string
+}
+
+var requestedParserFeatures = []apparmorParserFeature{
+	{"unsafe", "change_profile unsafe /**,"},
+}
+
+// tryParser will run the parser on the rule to determine if the feature is
+// supported.
+func tryParser(rule string) bool {
+	parser := "apparmor_parser"
+
+	_, err := exec.LookPath(parser)
```

Comment:
AFAICT this function could be simplified a bit:
```
func tryParser(rule string) bool {
	cmd := exec.Command("apparmor_parser", "--preprocess")
	cmd.Stdin = bytes.NewBufferString(fmt.Sprintf("profile snap-test {\n %s\n}", rule))
	if _, err := cmd.CombinedOutput(); err != nil {
		return false
	}
	return true
}
```
Because we don't return anything different if apparmor_parser does not exist or does not support the feature we can skip the LookupPath bit.

Reply:
Heh, this was what was originally there but @zyga wanted something explicit the lookup path :) I don't personally care either way ;)